From d4e8050a36c50ce8d55256995c6cbb4f9876c445 Mon Sep 17 00:00:00 2001
From: cs469 <cs469@cs469.cs.toronto.edu>
Date: Tue, 1 Oct 2019 22:45:18 -0400
Subject: [PATCH] generating patch for A1

---
 init/main.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/main.c b/init/main.c
index 96f8d5af52d6..fc5ece1d9db9 100644
--- a/init/main.c
+++ b/init/main.c
@@ -592,6 +592,7 @@ asmlinkage __visible void __init start_kernel(void)
 	 */
 	boot_cpu_init();
 	page_address_init();
+	printk(KERN_NOTICE "Welcome to sidhuj23 and brarjas8's kernel!");
 	pr_notice("%s", linux_banner);
 	setup_arch(&command_line);
 	mm_init_cpumask(&init_mm);
-- 
2.20.1

